### ISSUE TITLE
Typescript errors appearing out of nowhere, changing language mode back and forth makes them disappear

### ISSUE THREAD

**Issue by penrique** [Microsoft/vscode#29395]
- VSCode Version: Code 1.13.1 (379d2efb5539b09112c793d3d9a413017d736f89, 2017-06-14T18:21:47.485Z)
- OS Version: Windows_NT ia32 6.3.9600
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|html-snippets|abusaidm|0.1.0|
|vscode-css-formatter|aeschli|0.0.2|
|vscode-angular2-files|alexiv|1.3.8|
|Angular2|johnpapa|2.3.3|
|tsimporter|pmneo|1.2.14|
|vscode-icons|robertohuertasm|7.9.0|;

---

Steps to Reproduce:

1. Code
2. Errors suddenly appear even if there are none
3. How to fix temporarily:
4. Change language mode to something else
5. Switch back to Typescript and they will be gone
6. Keep coding and they will appear again
7. Go to Step 4 and repeat

This started happening 3 days ago not sure if update broke it

**Comment by mjbvz** [comment]
A few questions to help investigate:

* What error are you seeing?
* Can you share that code you are working with and some repo steps?
* Does this repo in all projects or just a specific one?
* Does this happen if you disable all extensions? Start code with the `--disable-extensions` command line flag to check this

Thanks

**Comment by penrique** [comment]
I realized It happened while working on exactly two files of my project. These files were at first created using the following names: Something.dynamic.ts and Something.dynamic.quadtree.ts. They were later renamed to something.dynamic.ts and something.dynamic.ts.
Things I noticed: 
- The integrated Git Source control did not notice the change.
- I was importing using import { Dynamic } from '.....Something.dynamic' instead of the new 'something.dynamic';
- Sometimes a second tab would open, one with 'something...' and the other one using 'Something....' which appeared as unsaved showing a grey dot on it.
- I managed to fix it by renaming the files back to Something.dynamic, etc...

Hope this helps
